packages/expandable-search-bar/package.json

```diff
@@ -32,25 +32,21 @@
     "test:prune-snapshots": "karma start --prune-snapshots",
     "test:compatibility": "karma start --compatibility all --coverage",
     "test:compatibility:watch": "karma start --compatibility all --auto-watch=true --single-run=false",
-    "test:bs": "karma start karma.bs.config.js --compatibility all --coverage",
-    "site:build": "npm run storybook:build",
-    "storybook": "start-storybook -p 9001",
-    "storybook:build": "build-storybook -o _site -s storybook-static"
+    "test:bs": "karma start karma.bs.config.js --compatibility all --coverage"
+    
```

Comment:
Could you remove the extra blank lines here and in `devDependencies`?

Reply:
It's done as per request. Any more changes?